xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -63,6 +63,21 @@ require(['jquery', 'jdateformatparser', 'daterangepicker', 'xwiki-events-bridge'
           updateInput(this, event, picker);
         });
 
+        
+        input.on('keypress', function (e) {
+          if(e.which !== 13) return;
```

Comment:
We always put the IF content in a block, even if it has just a single instruction, because it's less error prone. Also, you should leave a space after IF to follow the code style.